docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1429,6 +1429,24 @@ How it **works**:
 
 - The series must already exist before exemplars can be appended, as we do not create new series upon ingesting exemplars. The series will be created when a sample from it is ingested.
 
+### err-mimir-store-consistency-check-failed
+
+This error occurs when the querier is unable to fetch some of the expected blocks after multiple retries and connections to different store-gateways. The query fails because some blocks are missing.
+
```

Comment:
Let's try to explain how it works and how it could be fixed:

```suggestion

How it **works**:

- Mimir has been designed to guarantee query results correctness and never return partial query results. Either a query succeeds returning fully consistent results or it fails.
- Queriers and rulers when running with the "internal" evaluation mode run a consistency check to ensure all expected blocks have been queried from the long-term storage via store-gateways.
- If any expected block has not been queried via store-gateways, then the query fail with the `err-mimir-store-consistency-check-failed` error.
- See [Anatomy of a query request]({{< relref "../architecture/components/querier.md#anatomy-of-a-query-request" >}}) to learn more.

How to **fix** it:

- Ensure all store-gateways are healthy.
- Ensure all store-gateways are successfully synching owned blocks (see [`MimirStoreGatewayHasNotSyncTheBucket`](#MimirStoreGatewayHasNotSyncTheBucket)).

```